configure.ac

```diff
@@ -66,6 +66,11 @@ YES_IS_DEFINED
 dnl -----------------------------------------------
 dnl Check package options
 dnl -----------------------------------------------
+ AC_CHECK_HEADER([qemu/libvmi_request.h],
```

Comment:
This check should be moved below to the `[if test "$enable_kvm" = "yes"]` block.